### PR TITLE
added footer

### PR DESCRIPTION
**Pull request by miculnapoleon**

### CODE REVIEW

File: components/footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import Image from 'next/image'
+
+const FOOTER_ITEMS = [
+  {
+    alt: "Instagram",
+    icon:<a href="https://www.instagram.com/" target="__blank" className={styles.link}> <Image key={1}  width={20} height={20} src="/images/insta.svg" /> </a>
+  },
+  {
+    alt: "Behance",
+    icon: <a href="https://www.behance.net//" target="__blank" className={styles.linkB}> <Image key={2}  width={20} height={20} src="/images/behance.svg" /></a>,
```
Comment (MercurA) [inline components/footer/Footer.jsx]: link needs correction

Comment (MercurA) [reply components/footer/Footer.jsx]: also use `rel=”noopener noreferrer” ` for security resons

File: components/footer/footer.module.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+.iconContainer{
+    grid-row: 5;
+    grid-column: 1/5;
+    display: flex;
+    margin-left: 20px;
+}
+
+.link:hover {
+    fill:yellowgreen; }
```
Comment (MercurA) [inline components/footer/footer.module.scss]: format

**Review by MercurA — CHANGES_REQUESTED** [review]

File: components/footer/footer.module.scss (hunk 2 above)
Comment (miculnapoleon) [reply components/footer/footer.module.scss]: fixed

**Pull request merged**